README.md

```diff
@@ -314,32 +318,44 @@ Angle Regression
+* [rgexf](https://bitbucket.org/gvegayon/rgexf/wiki/Home) - Export network objects from R to [GEXF](http://gexf.net/format/), for manipulation with network software like [Gephi](https://gephi.github.io/) or [Sigma](http://sigmajs.org/).
+
 ## R Development
 *Packages for packages.*
 
 * [devtools](https://github.com/hadley/devtools) - Tools to make an R developer's life easier.
 * [testthat](https://github.com/hadley/testthat) - An R package to make testing fun.
 * [R6](https://github.com/wch/R6) - simpler, faster, lighter-weight alternative to R's built-in classes.
-* [pryr](https://github.com/hadley/pryr) -  Make it easier to understand what's going on in R.
+* [pryr](https://github.com/hadley/pryr) - Make it easier to understand what's going on in R.
 * [roxygen](https://github.com/klutometis/roxygen) - Describe your functions in comments next to their definitions.
 * [lineprof](https://github.com/hadley/lineprof) - Visualise line profiling results in R.
 * [packrat](https://github.com/rstudio/packrat) - Make your R projects more isolated, portable, and reproducible.
 * [installr](https://github.com/talgalili/installr/) - Functions for installing softwares from within R (for Windows).
-* [Rocker](https://github.com/rocker-org) - R configurations for Docker.
+* [Rocker](https://github.com/rocker-org) - R configurations for [Docker](https://www.docker.com/).
 * [drat](https://github.com/eddelbuettel/drat) - Creation and use of R repositories on GitHub or other repos.
-* [covr](https://github.com/jimhester/covr) - Test coverage for your R package and (optionally) upload the results to coveralls or codecov.
-* [lintr](https://github.com/jimhester/lintr) - Static code analysis for R to enforse code style.
+* [covr](https://github.com/jimhester/covr) - Test coverage for your R package and (optionally) upload the results to [coveralls](https://coveralls.io/) or [codecov](https://codecov.io/).
+* [lintr](https://github.com/jimhester/lintr) - Static code analysis for R to enforce code style.
 
-## Other Interpreter
+## Other Interpreters
 *Alternative R engines.*
 
 * [renjin](http://www.renjin.org/) - a JVM-based interpreter for R.
 * [pqR](http://www.pqr-project.org/) - a "pretty quick" implementation of R
-* [fastR](https://bitbucket.org/allr/fastr/wiki/Home) - FastR is an implementation of the R Language in Java atop Truffle and Graal. 
+* [fastR](https://bitbucket.org/allr/fastr/wiki/Home) - FastR is an implementation of the R Language in Java atop Truffle and Graal.
 * [riposte](https://github.com/jtalbot/riposte) - a fast interpreter and JIT for R.
 * [TERR](http://spotfire.tibco.com/discover-spotfire/what-does-spotfire-do/predictive-analytics/tibco-enterprise-runtime-for-r-terr) - TIBCO Enterprise Runtime for R.
 * [RRE](http://www.revolutionanalytics.com/) - Revolution R Enterprise.
-* [CXXR](https://www.cs.kent.ac.uk/projects/cxxr/) - Refactorising R into C++.
+* [CXXR](https://www.cs.kent.ac.uk/projects/cxxr/) - Refactorizing R into C++.

```

Comment:
Works for me. The document seemed primarily US English, so I harmonized based on that. My spellchecker prefers UK English anyway :)